### PR TITLE
WIP: Separate credentialing and training

### PR DESCRIPTION
**Pull request by fiedosiukr**
## Changes
- added separate tab in the user settings page - Trainings
- trainings and credentialing are separate processes now
- there are different training types with custom valid duration
- reviewer questions are customizable for each training type

## To do
- add a mechanism to allow project authors to select trainings that are required to access the data set

### CODE REVIEW

File: physionet-django/user/templates/user/edit_trainings.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{% extends "user/settings.html" %}
+
+{% block title %}PhysioNet Trainings{% endblock %}
```
Comment (alistairewj) [inline physionet-django/user/templates/user/edit_trainings.html]: `{{ SITE_NAME }} Training`?

Comment (fiedosiukr) [reply physionet-django/user/templates/user/edit_trainings.html]: Nice catch! It's been rebased after the debranding PR was merged and I forgot about that.

File: physionet-django/user/templates/user/edit_trainings.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          {% include 'user/training_badge.html' %}
+        </div>
+        <a href="{% url 'edit_trainings_detail' training.id %}" class="btn btn-sm btn-primary" role="button">View</a>
+      </li>
+    {% empty %}
+      <p>There are no trainings.</p>
+    {% endfor %}
+  </ul>
+  <hr>
+  <h4>New training</h4>
```
Comment (alistairewj) [inline physionet-django/user/templates/user/edit_trainings.html]: Submit evidence of a completed training course.

File: physionet-django/user/templates/user/edit_trainings.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  {% lorem 1 p %}
```
Comment (alistairewj) [inline physionet-django/user/templates/user/edit_trainings.html]: Provide a completion report indicating that you have finished the respective training course.

File: physionet-django/user/templates/user/edit_trainings.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{% extends "user/settings.html" %}
+
+{% block title %}PhysioNet Trainings{% endblock %}
+
+{% block main_content %}
+<h1>PhysioNet Trainings</h1>
+  <hr>
+  <h4>Active trainings</h4>
```
Comment (alistairewj) [inline physionet-django/user/templates/user/edit_trainings.html]: try to replace "trainings" with "training" wherever you see it

**Pull request merged**